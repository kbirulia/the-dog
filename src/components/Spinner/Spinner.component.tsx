import React from "react";
import "./Spinner.style.css";

const Spinner = (): JSX.Element => <div className="spinner" />;

export default Spinner;
