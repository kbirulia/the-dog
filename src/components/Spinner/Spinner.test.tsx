import React from "react";
import renderer from "react-test-renderer";
import Spinner from "./Spinner.component";

test("should render Spinner correctly", () => {
  const tree = renderer.create(<Spinner />).toJSON();
  expect(tree).toMatchSnapshot();
});
