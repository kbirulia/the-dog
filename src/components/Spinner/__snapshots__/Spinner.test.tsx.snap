// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Spinner correctly 1`] = `
<div
  className="spinner"
/>
`;
